meu_dict = {123: "um dois tres", "chave": "valor"}

print(meu_dict[123])
# print(meu_dict[chave]) com erro 
print(meu_dict["chave"])